File: translations/codenet/deepseek_commented/Go2P/gpt-4o/Code/s894183124.py
# Import the necessary module
import sys

# Read an integer `n` and a string `s` from the standard input
n = int(sys.stdin.readline().strip())
s = sys.stdin.readline().strip()

# Print the number of occurrences of the substring "ABC" in the string `s`
print(s.count("ABC"))

# <END-OF-CODE>

File: translations/codenet/gpt_commented/Go2C++/gpt-4o/Code/s894183124.cpp
#include <iostream>
#include <string>

int main() {
    // Declare variables to hold the integer input and the string input
    int n;
    std::string s;
    
    // Read an integer from standard input
    std::cin >> n;
    
    // Read a string from standard input
    std::cin >> s;
    
    // Count the occurrences of the substring "ABC" in the input string
    int count = 0;
    std::string substring = "ABC";
    size_t pos = s.find(substring);
    
    while (pos != std::string::npos) {
        count++;
        pos = s.find(substring, pos + 1);
    }
    
    // Print the result
    std::cout << count << std::endl;

    return 0;
}

// <END-OF-CODE>
